pyscript.core/src/stdlib/_pyscript/_display.py

```diff
@@ -0,0 +1,146 @@
+# ⚠️ WARNING - both `document` and `window` are added at runtime
+
+# XXX antocuni: I think this is wrong: it works in the main thread but not in
+# the worker, because the rest of the code expects window and document to be
+# proxies (see workerPyScriptModule in core.js)
```

Comment:
the test page shows already this works in worker ... it was tested in all browsers without issues but this file is an artifact, that's what the warning is about so I think now this breaks?

Reply:
It's not that it doesn't work, it's more that it basically does this in runtime (with eval) and makes it harder to manage these across different scopes.

Reply:
there's nothing to manage and everything in pyodide is evaluated out of files we write or code we provide ... I think we should just sit down today so I can explain to you how this works and why nothing in here improves anything, it actually breaks instead what's working already. We have other priorities so I hope we can avoid breaking working stuff before the release :sweat_smile: